app/src/main/java/com/andela/helpmebuy/models/PurchaseRequest.java

```diff
@@ -0,0 +1,33 @@
+package com.andela.helpmebuy.models;
+
+import java.util.ArrayList;
+
+public class PurchaseRequest extends Model{
+    private String sender;
+    private String receiver;
+    private ArrayList<PurchaseItem> purchaseList;
+
+    public String getSender() {
+        return sender;
+    }
+
+    public void setSender(String sender) {
+        this.sender = sender;
+    }
+
+    public String getReceiver() {
```

Comment:
From the explanation I got from the engineers, I was told the function is not in use for now thereby no need keeping it around @andela-ekarumbi